src/rnn.h

```diff
@@ -66,4 +66,4 @@ void compute_gru(const GRULayer *gru, float *state, const float *input);
 
 void compute_rnn(RNNState *rnn, float *gains, float *vad, const float *input);
 
-#endif /* _MLP_H_ */
+#endif /* _RNN_H_ */
```

Comment:
I am not able to evaluate the rest of the changes, but applied this one to get it out of the way.